microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
```

Comment:
Clarify the "description:" field.
`description: Force Microsoft Edge DevTools into Color Scheme Preview mode.`
At a glance, that description seems to imply that the article is about changing the theme of the DevTools portion of the window.  Is the article actually about changing the rendered webpage portion of the window instead?  It would be good to draw that distinction more explicitly.